intelmq/bin/intelmqctl.py

```diff
@@ -925,6 +926,7 @@ def check(self, no_connections=False, check_executables=True):
                                  "intelmqctl upgrade-config.")
 
         check_logger.info('Checking for bots.')
+        global_settings = files[RUNTIME_CONF_FILE]
```

Comment:
```suggestion
        global_settings = files[RUNTIME_CONF_FILE]['global']
```
Only the `global` key are global settings, not the whole file ;)